### DISCUSSION TITLE
Is it possibile to connect two recipes?

### DISCUSSION THREAD

**Discussion by pippo73** [TandoorRecipes/recipes#2631] [Q&A] [answered]
Hello, I would like to know if it is possible to connect 2 or more recipes.
For example if I cook lasagna, I know that I will need Béchamel sauce and Ragù. So I would like to connect them to the Lasagna recipe (with a link or somethink like that) in order to be able to access them quickly or just to know that I have them in the DB already...

thanks

**Comment by smilerz** [comment, marked as answer]
In your lasagna recipe you can add a step that is, itself, a recipe. 

![image](https://github.com/TandoorRecipes/recipes/assets/1808639/909c117d-3660-4e73-bf13-e62c1c4031ef)

**Reply by pippo73** [reply]
Thank you . I've not seen it :-)
